react/react installer/e257d740-d47d-4794-82f1-3704850bdba9.json

```diff
@@ -0,0 +1,56 @@
+{
```

Comment:
Most of the issues in the dependency installer repeat here, re. port types, serialization of `stdout`/`stderr`, etc.
Beyond that I feel the `success` output is redundant, as we either send something on `stdout` or `error`. If I need to turn that into a boolean, I can use a `boolean/Reverse condition` node.
One other thing I'd like to ask is to wrap this node into a structure node with an identical interface. As a general rule, we give preference to implementing nodes using existing nodes rather than JS, so this would keep the functionality of this node future-proof re. refactoring.